examples/monetization-with-stripe/pages/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from "react";
+import initMiro from "../initMiro";
+import { GetServerSideProps } from "next";
+import { storage } from "utils/storage";
+import { PAID_COOKIE_TITLE } from "pages/api/payment-handler";
+import { PaidFeature } from "components/PaidFeature";
+import { PaywallNotice } from "components/PaywallNotice";
+import { GenerallyAvailableFeature } from "components/GenerallyAvailableFeature";
+
+export const getServerSideProps: GetServerSideProps = async ({ req }) => {
+  const { userId, miro } = initMiro(req);
+
+  const hasPaid = (await storage.get(userId))?.[PAID_COOKIE_TITLE] || null;
+
+  // redirect to auth url if user has not authorized the app
+  if (!(await miro.isAuthorized(userId))) {
```

Comment:
nit: I think we're handling this differently in create-miro-app, might be worth using the same code here

Reply:
I started this example using CMA so i think its the same?

Reply:
I was thinking of the change we did recently in it, but ok to leave as is